payto/src/main/java/com/adyen/checkout/payto/internal/ui/view/PayToView.kt

```diff
@@ -71,6 +72,20 @@ internal class PayToView @JvmOverloads constructor(
         initBsbAccountNumberInput()
         initFirstNameInput()
         initLastNameInput()
+
+        updateInputFields(delegate.outputData)
```

Comment:
Shouldn't this be called before all the inputs are initialized? So it doesn't trigger listeners and creates this sort of loop